css/index.css

```diff
@@ -139,6 +139,86 @@ nav a {
     justify-content: center;
 }
 
+.boxes {
+    display: flex;
+    flex-wrap: wrap;
+    width: 800px;
+    height: 100px;
+    align-items: center;
+    padding: 8px;
+    justify-content: space-evenly;
+}
+
+
+.box1 {
+    background-color: teal;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box2 {
+    background-color: gold;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box3 {
+    background-color: cadetblue;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box4 {
+    background-color: coral;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box5 {
+    background-color: crimson;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box6 {
+    background-color: forestgreen;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box7 {
+    background-color: darkorchid;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box8 {
+    background-color: hotpink;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box9 {
+    background-color: indigo;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
+
+.box10 {
+    background-color: dodgerblue;
+    width: 100px;
+    height: 100px;
+    margin: 2%;
+}
```

Comment:
like we reviewed this can be consolidated to 1 box class and then a second class on all the boxes that will be used to simply change the color